client/src/components/CharityDetalis/Kpis/index.js

```diff
@@ -105,44 +176,71 @@ const Kpis = (props) => {
       <div className="kpi--row">
         <div className="kpi--column">
           <div className="kpi--item">
-            <span>Nnumber Of Trustees<p className="descrip-def">According to the Charity Governance code, a board of trustees of at least 5 but no more than 12 is typically good practice. </p></span>
-            <span>{numberOfTrustees}</span>
+            <span>Number Of Trustees</span>
+            <span>{returnFlag(numberOfTrustees)}</span>
+            <div className="overlayInner">
+            <p className="descrip-def innerDef">According to the Charity Governance code, a board of trustees of at least 5 but no more than 12 is typically good practice. </p>
+            </div>
           </div>
           <div className="kpi--item">
             <span>Executive Compensation</span>
-            <span>{executiveCompensation}</span>
+            <span>{returnFlag(executiveCompensation)}</span>
+            <div className="overlayInner">
+            <p className="descrip-def innerDef">According to the Charity Governance code, a board of trustees of at least 5 but no more than 12 is typically good practice. </p>
```

Comment:
@OnsNader I think this description doesn't match up with the information being displayed here (the description is about board of trustees, the field is executive compensation).  My apologies if this is still being finished up!

Reply:
There are a few fields below this that my comment applies to, also: policyHealth, trusteesBiography,